build/jest/base-jest-config.js

```diff
@@ -8,7 +8,10 @@
 
 /* eslint-env node */
 
-const testFolder = process.env.TEST_FOLDER || '__tests__';
+const matchField = process.env.TEST_REGEX ? 'testRegex' : 'testMatch';
+const matchValue = process.env.TEST_FOLDER
+  ? [`**/${process.env.TEST_FOLDER || '__tests__'}/**/*.js`]
```

Comment:
So we're removing the `__tests__` default folder, correct? Will this start running tests on files we had previously expected it not to? `__integration__` for example? My worry is that it might, and that we probably don't have any test cases to catch this.

Reply:
Default now uses TEST_MATCH with the same value as before. We do have tests w/ default config, as well as overridden testFolder (for integration), and now for overridden testMatch and testRegex.

The final jest config object under default fusion-cli configuration is still `testMatch: [**/__tests__/**/*.js]` as it was before

Reply:
Gotcha, didn't see that default setting in the CLI spec.